src/commands/import.js

```diff
@@ -0,0 +1,187 @@
+const fs = require('fs')
+const csv = require('csv-stream')
+const util = require('util')
+const { flags } = require('@oclif/command')
+const FaunaCommand = require('../lib/fauna-command.js')
+const FaunaWriteStream = require('../lib/fauna-write-stream')
+const faunadb = require('faunadb')
+const path = require('path')
+const q = faunadb.query
+
+class ImportCommand extends FaunaCommand {
+  async run() {
+    const { db } = this.flags
+    const { source } = this.args
+    const { client } = await (db
+      ? this.ensureDbScopeClient(db)
+      : this.getClient())
+    this.client = client
+
+    this.log(`Database${db ? `'${db}'` : ''} connection established`)
+
+    const isDir = fs.lstatSync(source).isDirectory()
+
+    return (isDir ? this.importDir() : this.importFile(source)).catch((error) =>
+      this.handleError(error)
+    )
+  }
+
+  async importDir() {}
+
+  async importFile(source) {
+    let { collection } = this.flags
+    const parsedFileName = this.parseFileName(source)
+    if (!collection) {
+      collection = parsedFileName.name
+    }
+    await this.dataImport({ source, collection })
+  }
+
+  parseFileName(source) {
+    const { name, ext } = path.parse(path.basename(source))
+
+    // TODO: support json
+    if (ext !== '.csv') {
+      throw new Error(`File (${source}) must include the '.csv' extension.`)
+    }
+    return { name, ext }
+  }
+
+  async dataImport({ source, collection }) {
+    const { ref, isCollectionExists } = await this.upsertCollection({
+      collection,
+    })
+
+    if (isCollectionExists) {
+      await this.clearCollectionDocuments(ref)
+    }
```

Comment:
I don't think we should clear the collection automatically. An unaware use might try to import _more_ data to a collection and ended deleting existing documents.